webapp/src/ts/components/duplicate-contacts/duplicate-contacts.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { NgIf } from '@angular/common';
+import { TranslatePipe } from '@ngx-translate/core';
+import { SimpleDatePipe } from '@mm-pipes/date.pipe';
+import {
+  MatExpansionPanel,
+  MatExpansionPanelHeader,
+  MatExpansionPanelTitle,
+  MatExpansionPanelDescription
+} from '@angular/material/expansion';
+import {
+  ContactSummaryContentComponent
+} from '@mm-components/contact-summary-content/contact-summary-content.component';
+import { Contact } from '@medic/cht-datasource';
+
+@Component({
+  selector: 'mm-duplicate-contacts',
+  templateUrl: './duplicate-contacts.component.html',
+  imports: [
+    NgIf,
+    TranslatePipe,
+    SimpleDatePipe,
+    MatExpansionPanel,
+    MatExpansionPanelHeader,
+    MatExpansionPanelTitle,
+    MatExpansionPanelDescription,
+    ContactSummaryContentComponent
+  ]
+})
+export class DuplicateContactsComponent {
+  @Input() duplicate: Contact.v1.Contact | undefined;
+  @Input() loadContactSummary: ((id: string) => Promise<void>) | undefined;
+  @Input() isLoading: boolean = false;
+  @Input() error?: string = undefined;
+  @Output() navigateToDuplicate = new EventEmitter<string>();
+
+  _summary;
+
+  _navigateToDuplicate(_id: string){
+    this.navigateToDuplicate.emit(_id);
+  }
+
+  async _loadContactSummary(duplicateId?: string){
+    if (!duplicateId){
+      return;
+    }
+    
+    if (this.isLoading){
+      return;
+    }
+
+    this.isLoading = true;
+    this.error = undefined;
+
+    try {
+      this._summary = await this.loadContactSummary?.(duplicateId);
+    } catch (err) {
+      this.error = `Unable to load summary data for contact ${duplicateId}`;
```

Comment:
issue: Can we load this message text from the translations?  (Also, I don't think the `duplicateId` will be super helpful to the user. They will already have the name/reported_date for the contact.)